styx-cli/src/main/java/com/spotify/styx/cli/Main.java

```diff
@@ -489,6 +537,25 @@ private void retryWorkflowInstance() throws ExecutionException, InterruptedExcep
     final Argument workflowShowWorkflowId =
         workflowShow.addArgument("workflow").help("Workflow ID");
 
+    final Subparser workflowCreate = WorkflowCommand.CREATE.parser(workflowParser);
+    final Argument workflowCreateComponentId =
+        workflowCreate.addArgument("component").help("Component ID");
+    final Argument workflowCreateFile =
+        workflowCreate.addArgument("-f", "--file")
+            .type(fileType().acceptSystemIn().verifyCanRead())
+            .help("Workflow configuration file");
+
+    final Subparser workflowDelete = WorkflowCommand.DELETE.parser(workflowParser);
+    final Argument workflowDeleteComponentId =
+        workflowDelete.addArgument("component").help("Component ID");
+    final Argument workflowDeleteWorkflowId =
+        workflowDelete.addArgument("workflow").help("Workflow ID");
+    final Argument workflowDeleteForce =
+        workflowDelete.addArgument("-f", "--force")
```

Comment:
`-f` means "force" in deletion but "file" in create? I'd be fine with having to type out `--force` here.

Reply:
👍